release-repo-scripts/bump_dependency.bash

```diff
@@ -57,6 +57,9 @@ VERSION_INPUT=${3}
 ISSUE_NUMBER=${4}
 PREV_COLLECTION=${5}
 DOCS_BRANCH=${6}
+if [[ -z "${DOCS_BRANCH}" ]]; then
+  DOCS_BRANCH=master
+fi
```

Comment:
It would probably be worth adding some documentation that `master` is the default branch if `DOCS_BRANCH` is empty. I think this could be mentioned after the `usage` docs: https://github.com/ignition-tooling/release-tools/blob/9f072b3adeda6e844f321e6d7a25d8bd0dfb6a42/release-repo-scripts/bump_dependency.bash#L21-L22

Reply:
https://github.com/ignition-tooling/release-tools/pull/557/commits/605085cbfb4de98f3c6d3ed7d07dac0be615954d